types/node/index.d.ts

```diff
@@ -2833,14 +2833,17 @@ declare module "fs" {
      */
     export function readFileSync(filename: string, options?: { flag?: string; }): Buffer;
     export function writeFile(filename: string, data: any, callback?: (err: NodeJS.ErrnoException) => void): void;
+    export function writeFile(filename: string, data: any, encoding: string, callback?: (err: NodeJS.ErrnoException) => void): void;
```

Comment:
Hmm. I noticed that `callback` is required in all versions for all overloads (no option, `encoding: string` and `options?: { ... }`)
Should these problems be fixed in this PR?